services/conda/conda-platform.service.js

```diff
@@ -1,27 +1,43 @@
 'use strict'
 
-const LegacyService = require('../legacy-service')
+const BaseCondaService = require('./conda-base')
 
-module.exports = class CondaPlatform extends LegacyService {
+module.exports = class CondaPlatform extends BaseCondaService {
   static get category() {
     return 'platform-support'
   }
 
   static get route() {
     return {
       base: 'conda',
+      pattern: ':which(p|pn)/:channel/:pkg',
     }
   }
 
   static get examples() {
     return [
       {
         title: 'Conda',
-        previewUrl: 'pn/conda-forge/python',
+        namedParams: { channel: 'conda-forge', package: 'python' },
+        pattern: 'pn/:channel/:package',
+        staticExample: this.render({
+          which: 'pn',
+          platforms: 'linux-64 | win-32 | osx-64 | win-64',
```

Comment:
The `|` concatenation feels like a rendering concern. What do you think about passing the array into `render()`?